helm/karpenter-crossplane-resources/values.schema.json

```diff
@@ -44,6 +44,19 @@
         },
         "serviceType": {
             "type": "string"
+        },
+        "internal": {
```

Comment:
Where would this come from?

Reply:
I've seen that value in the `{{ $.Values.clusterID }}-cluster-aws-cluster-values` and `{{ $.Values.clusterID }}-userconfig` ConfigMaps of migrated clusters. From my understanding that's generated by the migration-cli, but not 100% sure. Which led me to inject it like this: https://github.com/giantswarm/karpenter-bundle/pull/12

Do you know if I can get that irsa domain in another way?

Reply:
Addressed in the bundle PR: https://github.com/giantswarm/karpenter-bundle/pull/12/commits/fdc30e68a5c568ad8591da9e784a01b0ece8167b